netlib/tcp.py

```diff
@@ -369,14 +372,14 @@ def finish(self):
     """
 
     def _create_ssl_context(self,
-                            method=SSLv23_METHOD,
+                            method=TLSv1_2_METHOD,
```

Comment:
This also causes an issue with OpenSSL compatibility. TLS 1.2 was introduced in OpenSSL 1.0.1 (iirc). The default version on OSX 10.10.3 is is 0.9.8, so this will fail. I suggest we check what's supported on import and pick the best default to use.